HyperVExtension/src/HyperVExtension/Providers/HyperVProvider.cs

```diff
@@ -69,10 +75,21 @@ public IAsyncOperation<ComputeSystemsResult> GetComputeSystemsAsync(IDeveloperId
         {
             try
             {
+                if (!string.IsNullOrEmpty(_hyperVPreReqErrorText))
+                {
+                    // Dev Home contains code and error notifications to add the user to the admin group and enable Hyper-V.
+                    // So we do not need to send this error back to Dev Home as it will result in duplication.
+                    return new ComputeSystemsResult(new List<IComputeSystem>());
+                }
+
                 var computeSystems = _hyperVManager.GetAllVirtualMachines();
                 _log.Information($"Successfully retrieved all virtual machines on: {DateTime.Now}");
                 return new ComputeSystemsResult(computeSystems);
             }
+            catch (VirtualMachineManagementServiceException serviceException)
+            {
+                return new ComputeSystemsResult(serviceException, _stringResource.GetLocalized("HyperVVirtualMachineManagementServiceError"), serviceException.Message);
+            }
             catch (Exception ex)
             {
                 _log.Error(ex, $"Failed to retrieved all virtual machines on: {DateTime.Now}");

```

Comment:
Log inherently includes time, do we really need to put the time here too?

Reply:
....If the date logged wasn't `DateTime.Now` I would say keep.  But if the date is the same as the logging date I'd say get rid of it.

However, I can also vouch for the redundancy.  The DateTime in the log is when the line was logged.  Having a DateTime inside the message differentiates the log date vs. when the check happened.

Reply:
Good point. I looked and here's a real example of the success case from my logs:
`[2024/05/02 11:36:57.753 INF] (HyperVProvider) Successfully retrieved all virtual machines on: 5/2/2024 11:36:57 AM`

The time in the message is less precise than the time in the metadata, and if we made it as precise, do we care about the few millisecond difference?

Reply:
If I recall I think before the serilog change the logs were in UTC thats why I had these logged like this, but I can remove them